docs/poc2023q1/outline.md

```diff
@@ -0,0 +1,257 @@
+connectivity.  This depends on whether we can quickly and easily get a
+syncer that creates the appropriately independent objects in the edge
+cluster and itself tolerates intermittent connectivity.
+
+Also: initially this PoC will not transport non-namespaced objects.
+CustomResourceDefinitions are thus among the objects that will not be
+transported.  Transport of non-namespaced objects may be added after
+the other goals are achieved.
+
+As further matters of roadmapping development of this PoC:
+customization may be omitted at first, and summarization may start
+with only a limited subset of the implicit functionality.
+
+This PoC builds on TMC and makes some compromises to accommodate that.
+The implementation involves workload components (syncers) writing
+status information to inventory objects (SyncTargets).
+
+This PoC compromises on the connection initiative constraint in one
+place: the bootstrap step where central automation connects to edge
+clusters to install EMC machinery.
+
+## Design overview
+
+In very brief: the design is to reduce each edge placement problem to
+many instances of kcp's TMC problem.
+
+See [the overview picture](Edge-PoC-2023q1.svg) for an overview
+picture.
+
+## Inventory Management workspaces
+
+This design takes as a given that something maintains some kcp
+workspaces that contain dynamic collections of Location and SyncTarget
+objects as defined in [kcp
+TMC](https://github.com/kcp-dev/kcp/tree/main/pkg/apis), and that one
+view can be used to read those Location objects and one view can be
+used to read those SyncTarget objects.
+
+To enable the workload management layer to manipulate each edge
+cluster, that cluster's SyncTarget has an annotation that refers to a
+Secret that holds a kubeconfig that can be used by the Bootstrapper to
+install the edge-mc machinery in the edge cluster.  These annotations
+and Secrets are created by the inventory management layer for use by
+the workload management layer.
+
+To complete the plumbing of the syncers, for each SyncTarget the
+mailbox controller creates the following needed accompanying items in
+that SyncTarget's workspace (which must authorize the mailbox
+controller to do this).
+
+1. A ServiceAccount that the syncer will authenticate as.
+2. A ClusterRole manipulating that SyncTarget and the
+   APIResourceImports (what are these?).
+3. A ClusterRoleBinding that links that ServiceAccount with that
+   ClusterRole.
+
+FYI, those are the things that `kubectl kcp workload sync` directly
+creates besides the SyncTarget.
+
+## Edge Service Provider workspace
+
+The edge multi-cluster service is provided by one workspace that
+includes the following things.
+
+- An APIExport of the edge API group.
+- The edge controllers: scheduler, placement translator, bootstrapper,
+  mailbox controller, and status sumarizer.
+
+## Workload Management workspaces
+
+The users of edge multi-cluster primarily maintain these.  Each one
```

Comment:
What do you mean by "users of edge multi-cluster"  ? is it the admins of the edge cluster? users of the edge? admins of the edge-mc platform?  We probably need to define roles in the PoC